### DISCUSSION TITLE
custom jwt problem

### DISCUSSION THREAD

**Discussion by BoE1008** [Web3Auth/Web3Auth#1369] [Help]
this is my code 
![screenshot-20230224-120106](https://user-images.githubusercontent.com/29189493/221088948-3aba269f-1c5e-4e77-a6f1-0d4e14244091.png)

the error
![image](https://user-images.githubusercontent.com/29189493/221089094-ab612070-217a-40f3-abc2-920a98b98fef.png)

The configuration items are correct, but returned 404

**Comment by ugoarzur** [comment]
Hi,
It seems you are instanciating with the `testnet` mode but the error says to switch the network to `mainnet`. You may want to check your project network [in your web3auth dashboard](https://dashboard.web3auth.io/) within the "plug and play" section in to verify that your `clientId` is associated to a project in `testnet` and not in `mainnet`.
